From martinez.javier@gmail.com  Thu Aug 12 13:05:33 2010
Subject: Staging: use new REQ_FLUSH flag, fix build breakage
From: Javier Martinez Canillas <martinez.javier@gmail.com>
To: Greg KH <greg@kroah.com>, Randy Dunlap <randy.dunlap@oracle.com>,  devel@driverdev.osuosl.org, David Woodhouse <David.Woodhouse@intel.com>,  kernel-janitors@vger.kernel.org
Date: Fri, 09 Jul 2010 23:28:13 -0400
Message-ID: <1278732493.1725.19.camel@lenovo>

REQ_TYPE_LINUX_BLOCK and REQ_LB_OP_FLUSH doesn't exist anymore. Using
the new REQ_FLUSH flag instead

Signed-off-by: Javier Martinez Canillas <martinez.javier@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/spectra/ffsport.c |    3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

--- a/drivers/staging/spectra/ffsport.c
+++ b/drivers/staging/spectra/ffsport.c
@@ -289,8 +289,7 @@ static int do_transfer(struct spectra_na
 			IdentifyDeviceData.PagesPerBlock *
 			res_blks_os;
 
-	if (req->cmd_type == REQ_TYPE_LINUX_BLOCK &&
-			req->cmd[0] == REQ_LB_OP_FLUSH) {
+	if (req->cmd_type & REQ_FLUSH) {
 		if (force_flush_cache()) /* Fail to flush cache */
 			return -EIO;
 		else
